x/ccv/provider/client/proposal_handler.go

```diff
@@ -441,3 +452,29 @@ func postEquivocationProposalHandlerFn(clientCtx client.Context) http.HandlerFun
 		tx.WriteGeneratedTxResponse(clientCtx, w, req.BaseReq, msg)
 	}
 }
+
+func CheckPropUnbondingPeriod(clientCtx client.Context, propUnbondingPeriod time.Duration) error {
+	client, err := clientCtx.GetNode()
+	if err != nil {
+		return err
+	}
+
+	consensusParams, err := client.ConsensusParams(context.Background(), nil)
+	if err != nil {
+		return err
+	}
+
+	providerUnbondingTime := consensusParams.ConsensusParams.Evidence.MaxAgeDuration
```

Comment:
Not sure about this, maxAgeDuration is not the same as the unbonding period. The evidence age is configured separately from the unbonding period, those are 2 different parameters.

Reply:
You are right, has been adjusted